src/RcppIRV.cpp

```diff
@@ -214,10 +228,13 @@ void PIRVDirichletTree::update(Rcpp::List ballots) {
     // the minDepth of the tree, otherwise the posterior tree
     // will no longer be reducible to a Dirichlet distribution.
     depth = bc.first.nPreferences();
-    if (depth < minDepth && tree->getParameters()->getVD())
-      Rcpp::warning("Updating a Dirichlet distribution with a ballot "
-                    "specifying fewer than `minDepth` preferences. The "
-                    "resulting posterior is no longer Dirichlet.");
+    if (depth < minDepth)
```

Comment:
@dvukcevic The change from `if (depth < minDepth && tree->getParameters()->getVD())` to `if (depth < minDepth)` is the cause of the new warnings. I propose we change it to `if (0 < depth < minDepth)`.

Reply:
Sounds like a good idea, @fleverest

Reply:
I've implemented this now.